### PR TITLE
VACMS-1421: Replace non rich text intro field with rich text intro.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #1421 
**DO NOT MERGE PRIOR TO https://github.com/department-of-veterans-affairs/vets-website/pull/15880 (merged) being released to prod.**

## Testing done
Visual / Behat

## Screenshots
image

## QA steps
As an administrator, go to `/node/993/edit` and visually verify that the page introduction field is wysiwyg

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ethanteague** [comment]
@swirtSJW I'll update the web version again tomorrow

File: tests/behat/drupal-spec-tool/content_model_benefits_hubs_content_type_fields.feature [hunk 1]
```diff
@@ -4,7 +4,7 @@ Feature: Content model: Benefits hubs Content Type fields
   As a content editor
   I want to have content type fields that reflect my content model.
 
-  @dst @field_type @content_type_fields @dstfields @boo
```
Comment (ndouglas) [inline tests/behat/drupal-spec-tool/content_model_benefits_hubs_content_type_fields.feature:7]: 👻

File: config/sync/core.entity_form_display.node.page.default.yml [hunk 2]
```diff
@@ -77,6 +77,7 @@ third_party_settings:
       children:
         - title
         - field_intro_text
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.page.default.yml:79]: Shouldn't this line be going away?

File: config/sync/core.entity_form_display.node.page.default.yml [hunk 3]
```diff
@@ -203,19 +204,14 @@ content:
...
-  field_intro_text:
-    weight: 3
+  field_intro_text_limited_html:
+    type: text_textarea
+    weight: 2
+    region: content
     settings:
       rows: 5
       placeholder: ''
-      maxlength: 600
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.page.default.yml:211]: Shouldn't this length limit and counter be applied to the new field (field_intro_text_limited_html)?

File: config/sync/core.entity_form_display.node.page.default.yml (hunk 2 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.node.page.default.yml]: Good catch, @swirtSJW - removed.

File: config/sync/core.entity_form_display.node.page.default.yml (hunk 3 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.node.page.default.yml:211]: I don't think so - it's a rich text field, whereas field_intro_text was not, @swirtSJW - maybe @kevwalsh can weigh in.

Comment (swirtSJW) [reply config/sync/core.entity_form_display.node.page.default.yml:211]: Discussed in 16th min.  Counter not needed.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**